script/verify-changes.test.ts

```diff
@@ -0,0 +1,21 @@
+import fs from "fs/promises";
+import { expect } from "vitest";
+import { test } from "vitest";
+
+test("knip.jsonc", async () => {
```

Comment:
[Refactor] This is a great start! +1 that this test catches the specific properties that are different now.

But: it only catches these specific properties. If the template were to be updated in the future with some other properties, this test wouldn't catch them. For example, let's say a PR adds a `"someAwesomeProperty": true` to the 🍎  _template_ but not the 🍌 `knip.jsonc` file. This test wouldn't fail. But it should fail: it's meant to catch any mismatch between the 🍎 template and the 🍌 file.

I think a more future-proof strategy would be to assert on the actual difference between the 🍎 template and the 🍌 file. How about checking for what the `git diff` output is back in `migrate-test-e2e.js`? The file already has sample code running `git diff HEAD -- ...`.

Reply:
I see, that makes sense! I'm afraid with `git diff HEAD -- ...` it's hard to assert the changes that for instance add a new item to the config array. 

Can we do it through Object.entries or Object.keys also for instance? 

Sorry for the late reply.

Reply:
Looks like you figured it out in a later commit!